packages/rocketchat-logger/client/views/viewLogs.coffee

```diff
@@ -96,3 +97,8 @@ Template.viewLogs.onRendered ->
 		Meteor.setTimeout ->
 			template.checkIfScrollIsAtBottom()
 		, 2000
+
+	wrapper.addEventListener 'scroll', ->
+		template.atBottom = false
+		Meteor.defer ->
+			template.checkIfScrollIsAtBottom()
```

Comment:
This will execute the check many times, can you use `_.throttle` like we use [here](https://github.com/edwardmuel/Rocket.Chat/blob/870d42152c00a05fc3e2eec3dcabe86526d26de5/packages/rocketchat-ui/views/app/room.coffee?ts=2#L321) ?